numba/__init__.py

```diff
@@ -246,3 +246,9 @@ def _try_enable_svml():
 # SVML state to "no SVML". See https://github.com/numba/numba/issues/4689 for
 # context.
 # ---------------------- WARNING WARNING WARNING ----------------------------
+
+
+def load_ipython_extension(ip):
```

Comment:
Is this to support `%load_ext numba`? If so, could you add a comment mentioning this, for the benefit of readers unfamiliar with how magics work? (They are new to me, and I imagine others might also benefit from the explanation)